src/tests/ParamTest.cs

```diff
@@ -25,21 +25,25 @@ public async Task AppMainTest ()
 
             args = new string[] { "-k", "heliumtest-kv", "-a", "FOO" };
             i = await App.Main(args);
-            Assert.Equal(1, i);
+            Assert.NotEqual(0, i);
 
-            Assert.Equal(2, App.CombineEnvVarsWithCommandLine(null).Count);
+            Assert.Equal(4, App.CombineEnvVarsWithCommandLine(null).Count);
 
```

Comment:
(For my understanding) - why is this 4?

Reply:
because we added --log-level which creates 2 entries (--log-level and value)